### ISSUE TITLE
Can't remove SC events subscriptions for de-provisioned endpoints

### ISSUE THREAD

**Issue by SeanFeldman** [Particular/ServiceControl#946]
Currently, there's no way to remove SC events subscriptions from RavenDB w/o manually tweaking the database. There's no guidance or a tool/script to assist with that either.

**Comment by mikeminutillo** [comment]
I know @SzymonPobiega had a plan to configure event subscribers in the SC config file and turning the emission of these events into Sends (rather than publishes).

**Comment by SzymonPobiega** [comment]
@mikeminutillo that's true. But in order to do this I think we would have to figure out a proper migration plan to make sure customers who have critical functionality depending on these events are not left alone.

**Comment by fourpastmidnight** [comment]
Provided there was an API to call via #1561 when deprovisioning endpoints, wouldn't it make sense, as part of the deprovision process, to allow one to specify that event subscriptions should be removed when the endpoint is deprovisioned?

**Comment by SzymonPobiega** [comment]
Considering how the event notifications work (via pub/sub) I think this is very difficult to achieve cross-transport as most transports now support native pub sub so the operation of unsubscribing can't really be conducted from SC point of view.
